classes/apiManager.js

```diff
@@ -0,0 +1,33 @@
+class ApiManager {
+    constructor() {
+        this.team = []
+    }
+
+    getTeam(teamName) {
+        return $.get(`/teams/${teamName.toLowerCase()}`, data => {
```

Comment:
there's no need to use the 'return' - this function just sending a get request to the server and updates this.team - you don't use the promise in main.js